ci/packet/packet-cluster.tf.envsubst

```diff
@@ -0,0 +1,62 @@
+module "controller" {
+  source = "../../packet/flatcar-linux/kubernetes"
+
+  providers = {
+    aws      = "aws.default"
+    local    = "local.default"
+    null     = "null.default"
+    template = "template.default"
+    tls      = "tls.default"
+    packet   = "packet.default"
+  }
+
+  dns_zone = "$AWS_DNS_ZONE"
+  dns_zone_id = "$AWS_DNS_ZONE_ID"
+
+  ssh_keys = ["$PUB_KEY"]
+
+  asset_dir = "${pathexpand("~/assets")}"
+  cluster_name = "$CLUSTER_ID"
+  project_id = "$PACKET_PROJECT_ID"
+
+  facility = "sjc1"
```

Comment:
We can use locals to avoid duplicating this here and in the worker module (or env vars, as we prefer)